### PR TITLE
WOTLK: Updating Potion Recommendations

### PR DESCRIPTION
**Pull request by TJMoats**
<details>
<summary>With Potion Use</summary>

f09SJbW
</details>
<details>
<summary>Without Potion Use</summary>

JaPDnJC
</details>

### CODE REVIEW

File: src/parser/classic/modules/items/Potion.ts [hunk 1]
```diff
@@ -1,100 +0,0 @@
...
- * There are three different categories of pots that share cooldown:
- * Healthstones, health pots and combat pots (DPS, HPS, mana and mitigation).
- * All potions have a 5 minute cooldown.
- *
- * @property {Abilities} abilities
- * @property {Auras} buffs
- * @property {SpellUsable} spellUsable
- * @property {AbilityTracker} abilityTracker
- */
-class Potion extends Analyzer {
```
Comment (TJMoats) [inline src/parser/classic/modules/items/Potion.ts:21]: Classic and retail have the same setup for potions now, so there is no reason to maintain this custom class.

**Comment by Pewtro** [comment]
seems fine once you fix the merge conflict

**Review by Pewtro — APPROVED** [review]

**Comment by TJMoats** [comment]
I can't for the life of me get this pr to update. My branch has the conflicts resolved, but the pr isn't updating to track it.

**Comment by TJMoats** [comment]
Closed in favor of https://github.com/WoWAnalyzer/WoWAnalyzer/pull/5248

**Pull request closed without merging**